dapp/e2e/register.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { test, expect } from '@playwright/test'
+
+import type { CheckResponse } from './utils';
+import{
+    checkString,
+    checkImage,
+    checkButton,
+    checkInput,
+    checkColor,
+    checkLabel,
+} from './utils'
+
+
+import { getDemoConfigData } from "../src/constants/demoConfigData";
+const data = getDemoConfigData();
+
+test.describe("Register page test", () => {
+  
+    test.beforeAll(async () => {
+      console.log("Before tests");
+    });
+  
+    test.afterAll(async () => {
+      console.log("After tests");
+    });
+  
+    // Check the Url is correct!
+    test("correct Url", async ({ page }) => {
+        // ✅ Navigate to the URL first
+        await page.goto("http://localhost:4321/register");
+        
+        // ✅ Then, check if the URL is correct
+        await expect(page).toHaveURL("http://localhost:4321/register");
+        
+        console.log("Test passed!");
+    });
+    
+    // Check home UI is correct!
+    test("home Ui test", async({page}) => {
+        const results : CheckResponse[] = [];
+    
+        await page.goto("http://localhost:4321/register");
+    
+        // Check if Images exsit
+        results.push(await checkImage(page, undefined, "Logo", "Logo"));
+        results.push(await checkImage(page, undefined, "(//)", "(//)"));
+        results.push(await checkImage(page, "footer-github"));
+        results.push(await checkImage(page, "footer-stellar"));
+        results.push(await checkImage(page, "footer-discord"));
+    
+        // Check Labels exist
+        results.push(await checkLabel(page, undefined, "project"));
+        results.push(await checkLabel(page, undefined, "Maintainers"));
+        results.push(await checkLabel(page, undefined, "GitHub repository URL"));
+        results.push(await checkLabel(page, undefined, "Information file hash"));
+        
+        // Check general Strings exist
```

Comment:
A general comment. I prefer if we focus first on checking features. The design is going to change and this is ok as long as we ensure the App still works as intended.